Benchmarks/Program.fs

```diff
@@ -0,0 +1,77 @@
+﻿namespace Benchmarks
+
+open System
+open BenchmarkDotNet.Attributes
+open BenchmarkDotNet.Running
+open BenchmarkDotNet.Configs
+open BenchmarkDotNet.Reports
+open BenchmarkDotNet.Columns
+open BenchmarkDotNet.Environments
+open Perfolizer.Horology
+
+open ArraySorts
+open MyListSorts
+
+
+[<MemoryDiagnoser>]
+type MyBenchmark() =
+    
+    static member Lengths = [|10000..10000..100000|]
+    member this.Random = System.Random()
+
+    
+    // [<ParamsSource("Lengths")>]
+    // member val ArrayLength = 0 with get, set
+    // member val ArrayToSort = [|0.0|] with get, set
+
+    // [<IterationSetup>]
+    // member this.GetArrayToSort () = 
+    //     this.ArrayToSort <- Array.init this.ArrayLength (fun _ -> this.Random.NextDouble()) 
+
+
+    [<ParamsSource("Lengths")>]
+    member val ListLength = 0 with get, set
+    member val ListToSort = [0.0] with get, set
+
+    [<IterationSetup>]
+    member this.GetListToSort () = 
+        this.ListToSort <- List.init this.ListLength (fun _ -> this.Random.NextDouble()) 
+
+
+    // [<Benchmark>]

```

Comment:
You can have several benchmarks and run only some of them by carefully configured entrypoint.